portfolio/src/main/webapp/me.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
```

Comment:
(nit) https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity has `utf-8`in lowercase